src/commands/api/DatabaseAccountTreeItemInternal.ts

```diff
@@ -58,6 +59,12 @@ export class DatabaseAccountTreeItemInternal implements DatabaseAccountTreeItem
         ext.treeView.reveal(await this.getAccountNode());
     }
 
+    public async openInPortal(resourceID: string): Promise<void> {
+        if (this._accountNode) {
```

Comment:
If you don't have the accountNode, I think you should call getAccountNode then rather than just silently failing.